lib/msprime.h

```diff
@@ -336,14 +337,21 @@ typedef struct {
 } infinite_sites_mutation_t;
 
 typedef struct {
-    int alphabet;
+    int alphabet_length;
```

Comment:
Usually make these ``size_t`` or the like. Avoids casting to unsigned on mallocs.